include/mbedtls/ssl.h

```diff
@@ -2533,13 +2533,19 @@ void mbedtls_ssl_conf_cert_req_ca_list( mbedtls_ssl_config *conf,
 #if defined(MBEDTLS_SSL_MAX_FRAGMENT_LENGTH)
 /**
  * \brief          Set the maximum fragment length to emit and/or negotiate
- *                 (Default: the smaller of MBEDTLS_SSL_IN_CONTENT_LEN and
+ *                 (Typical: the smaller of MBEDTLS_SSL_IN_CONTENT_LEN and
  *                 MBEDTLS_SSL_OUT_CONTENT_LEN, usually 2^14 bytes)
  *                 (Server: set maximum fragment length to emit,
- *                 usually negotiated by the client during handshake
+ *                 usually negotiated by the client during handshake)
  *                 (Client: set maximum fragment length to emit *and*
  *                 negotiate with the server during handshake)
  *
+ * \note           By default the \c mfl_code field of the \c mbedtls_ssl_config
+ *                 structure is equal to `0 == MBEDTLS_SSL_MAX_FRAG_LEN_NONE`.
```

Comment:
I find the `0 == MBEDTLS_SSL_MAX_FRAG_LEN_NONE` confusing (does it mean that it's 0, or that it's the result of evaluating the expression). I'd remove the `0 == ` par and just go with the named constant.